apport: report to the "pantheon-dock" dummy project
data: add NotShowIn=Pantheon; to desktop-file

=== modified file 'data/apport/plank-crashdb.conf'
--- old/data/apport/plank-crashdb.conf	2012-03-09 21:45:40 +0000
+++ new/data/apport/plank-crashdb.conf	2015-11-07 07:42:02 +0000
@@ -1,5 +1,5 @@
 plank = {
 	'impl' : 'launchpad',
-	'project' : 'plank',
+	'project' : 'pantheon-dock',
 	'bug_pattern_base' : None,
 }

=== modified file 'data/plank.desktop.in'
--- old/data/plank.desktop.in	2012-10-09 16:37:49 +0000
+++ new/data/plank.desktop.in	2015-11-07 07:42:03 +0000
@@ -8,3 +8,4 @@ Exec=plank
 Icon=plank
 Terminal=false
 NoDisplay=false
+NotShowIn=Pantheon;

